src/cascadia/TerminalApp/TabBase.cpp

```diff
@@ -252,4 +254,291 @@ namespace winrt::TerminalApp::implementation
         });
     }
 
+    std::optional<winrt::Windows::UI::Color> TabBase::GetTabColor()
+    {
+        return std::nullopt;
+    }
+
+    void TabBase::ThemeColor(const winrt::Microsoft::Terminal::Settings::Model::ThemeColor& focused,

```

Comment:
note to reviewers: This mostly just came from `TerminalTab.cpp` so go ahead and hit that <kbd>.</kbd> and open the two SxS
![image](https://user-images.githubusercontent.com/18356694/187669910-0c02d739-3822-4ae1-bbd2-2616a1952c8e.png)